utils.py

```diff
@@ -117,6 +117,8 @@ def _read_builds():
     if os.environ.get("MOCK"):
         builds = "mock.builds.json"
     try:
+        if pathlib.Path(builds).stat().st_size == 0:
+            raise Exception("%s is zero sized?" % (builds))
```

Comment:
do you want to use an `f` string here?

Reply:
Oops, yes! This was the thing that made me ask about the conversions in the first place so this should definitely be updated.

Reply:
49e96e7a5b8e72b9cccf8a9b2becca18efd6cc78